### PR TITLE
adjusted form layout and inputs

### PR DESCRIPTION
**Pull request by CamillieFu**
Adjust form layout and inputs so we not have a dropdown for styles and the min input for integers is 1 or 0(for size
Screen Shot 2022-01-28 at 10 07 58
)
Screen Shot 2022-01-28 at 10 09 23

### CODE REVIEW

File: app/views/bookings/_form.html.erb [hunk 1]
```diff
@@ -1,10 +1,7 @@
     <%= simple_form_for [ @dress, @booking ] do |f| %>
       <%= f.input :starting_date, as: :string, required: false, input_html: {id: "start_date"} %>
       <%= f.input :ending_date, as: :string, required: false, input_html: {id: "end_date"} %>
-      <%= f.submit class: 'btn btn-outline-secondary float-right'%>
-      <p id= day_price>Price per day: $<%= @dress.price_per_day %></p>
-      <div style="display:flex;"><p>Total: $</p><p id="price">0</p>
-      </div>
```
Comment (angrysun) [inline app/views/bookings/_form.html.erb:7]: @CamillieFu I believe this is the new code that @Gazwai just added last night. I think we should keep this.

**Comment by angrysun** [comment]
Very nice!

File: app/views/bookings/_form.html.erb (hunk 1 above)
Comment (CamillieFu) [reply app/views/bookings/_form.html.erb:7]: The code doesn't show any total for the moment - we need to use Javascript - I think I left it in a comment?

Comment (CamillieFu) [reply app/views/bookings/_form.html.erb:7]: I can leave it if you'd like though! But it just shows the price per day of the dress and an empty total ATM

Comment (angrysun) [reply app/views/bookings/_form.html.erb:7]: Oh yea you can comment it out. I just don't want to delete Gary's hard work lol.

Comment (CamillieFu) [reply app/views/bookings/_form.html.erb:7]: OK!

**Pull request merged**